backend/src/test/resources/search.sql

```diff
@@ -0,0 +1,98 @@
+VALUES (2, '2024-09-27 08:43:08.763888', false, 1, '2024-09-27 08:43:08.763888', 'Category 2');
+
+-- Tag 삽입
+INSERT INTO tag (id, created_at, modified_at, name)
+VALUES (1, '2024-09-27 08:43:08.767154', '2024-09-27 08:43:08.767154', 'Tag 1');
+
+INSERT INTO tag (id, created_at, modified_at, name)
+VALUES (2, '2024-09-27 08:43:08.769440', '2024-09-27 08:43:08.769440', 'Tag 2');
+
+-- Template 삽입
+INSERT INTO template (id, category_id, created_at, description, member_id, modified_at, title)
+VALUES (1, 1, '2024-09-27 08:43:08.773369', 'Description 1', 1, '2024-09-27 08:43:08.773369', 'Template 1');
+
+INSERT INTO template (id, category_id, created_at, description, member_id, modified_at, title)
+VALUES (2, 2, '2024-09-27 08:43:08.787995', 'Description 1', 1, '2024-09-27 08:43:08.787995', 'Template 1');
+
+INSERT INTO template (id, category_id, created_at, description, member_id, modified_at, title)
+VALUES (3, 1, '2024-09-27 08:43:08.790778', 'Description 1', 2, '2024-09-27 08:43:08.790778', 'Template 1');
+
+-- Source Code 삽입
+INSERT INTO source_code (id, content, created_at, filename, modified_at, ordinal, template_id)
+VALUES (1, 'content1', '2024-09-27 08:43:08.793152', 'filename1', '2024-09-27 08:43:08.793152', 1, 1);
+
+INSERT INTO source_code (id, content, created_at, filename, modified_at, ordinal, template_id)
+VALUES (2, 'content2', '2024-09-27 08:43:08.797349', 'filename2', '2024-09-27 08:43:08.797349', 2, 2);
+
+INSERT INTO source_code (id, content, created_at, filename, modified_at, ordinal, template_id)
+VALUES (3, 'content1', '2024-09-27 08:43:08.810641', 'filename1', '2024-09-27 08:43:08.810641', 1, 3);
+
+-- Thumbnail 삽입
+INSERT INTO thumbnail (id, created_at, modified_at, source_code_id, template_id)
+VALUES (1, '2024-09-27 08:43:08.812840', '2024-09-27 08:43:08.812840', 1, 1);
+
+INSERT INTO thumbnail (id, created_at, modified_at, source_code_id, template_id)
+VALUES (2, '2024-09-27 08:43:08.815866', '2024-09-27 08:43:08.815866', 2, 2);
+
+INSERT INTO thumbnail (id, created_at, modified_at, source_code_id, template_id)
+VALUES (3, '2024-09-27 08:43:08.817850', '2024-09-27 08:43:08.817850', 3, 3);
+
+-- Template_Tag 삽입
+INSERT INTO template_tag (created_at, modified_at, tag_id, template_id)
+VALUES ('2024-09-27 08:43:08.835179', '2024-09-27 08:43:08.835179', 1, 1);
+
+INSERT INTO template_tag (created_at, modified_at, tag_id, template_id)
+VALUES ('2024-09-27 08:43:08.843467', '2024-09-27 08:43:08.843467', 2, 1);
+
+INSERT INTO template_tag (created_at, modified_at, tag_id, template_id)
+VALUES ('2024-09-27 08:43:08.851712', '2024-09-27 08:43:08.851712', 1, 2);
+
+INSERT INTO template_tag (created_at, modified_at, tag_id, template_id)
+VALUES ('2024-09-27 08:43:08.860091', '2024-09-27 08:43:08.860091', 2, 2);
+
+INSERT INTO template_tag (created_at, modified_at, tag_id, template_id)
+VALUES ('2024-09-27 08:43:08.867619', '2024-09-27 08:43:08.867619', 2, 3);
+
+-- 전문 검색 인덱스가 존재하는지 조회
```

Comment:
정말정말 사소하지만, 주석으로 정보를 제공해 준다면 최대한 간결하게 중요한 정보를 다 전달해 줄 수 있으면 좋겠어요!!!
```suggestion
-- 템플릿 테이블에 전문 검색 인덱스가 존재하는지 조회
```

Reply:
추가했습니다~!

Reply:
몰체~~~~